nengo_gui/static/nengo.js

```diff
@@ -72,13 +72,17 @@ Nengo.next_zindex = function() {
 }
 
 /* draw a legend */
-// the css should probably be dealt with in here somehow
-Nengo.draw_legend = function(parent, labels, color_func){
+Nengo.draw_legend = function(parent, labels, color_func, uid) {
     // "20" is around the size of the font
     legend_svg = d3.select(parent)
                        .append("svg")
                        .attr("width", 150)
-                       .attr("height", 20*labels.length);
+                       .attr("height", 20*labels.length)
+                       .attr("id", "legend"+uid);
+
+    if(labels.length == 0) {
```

Comment:
The reason that I keep missing stuff like this is because I was using
jshint to detect style stuff, when apparently it only does functionality
stuff. Style checking happens with jscs apparently... Fixed now.

On Tue, Nov 17, 2015 at 5:17 PM, tcstewar <notifications@github.com> wrote:

> In nengo_gui/static/nengo.js
> <https://github.com/nengo/nengo_gui/pull/610#discussion_r45130635>:
>
> >      // "20" is around the size of the font
> >      legend_svg = d3.select(parent)
> >                         .append("svg")
> >                         .attr("width", 150)
> > -                       .attr("height", 20*labels.length);
> > +                       .attr("height", 20*labels.length)
> > +                       .attr("id", "legend"+uid);
> > +
> > +    if(labels.length == 0) {
>
> spacing! :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nengo/nengo_gui/pull/610/files#r45130635>.
>